docs/language-server/protocol-language-server.md

```diff
@@ -2740,6 +2796,61 @@ null;
 - [`WriteDeniedError`](#writedeniederror) to signal that the client doesn't hold
   write lock for the buffer.
 
+### `text/applyExpressionValue`
+
+This requests to apply specific value to the expression. I.e. it can be an
+update of a literal value, like changing `98` to `99`, `true` to `false` or
+`"Hello"` to `"World!"`. This method is a more specific version of
```

Comment:
```suggestion
This requests to set an expression to a new value. For example,
it can update a literal value, like changing `98` to `99`, `true` to `false` or
`"Hello"` to `"World!"`. This method is a more specific version of
```